Userland/Applets/Audio/main.cpp

```diff
@@ -24,6 +24,8 @@
 #include <LibGfx/Palette.h>
 #include <LibMain/Main.h>
 
+static constexpr bool DEFAULT_AudioApplet_Applet_ShowPercent = false;
```

Comment:
I get why you named this variable this way, but it diverges from our code style. Let's just name it `audio_applet_show_percent_default` for now.